components/common/AudioPlayer.js

```diff
@@ -55,15 +24,11 @@ AudioPlayer.propTypes = {
   trackId: PropTypes.string.isRequired,
   width: PropTypes.string,
   height: PropTypes.string,
-  color: PropTypes.string,
-  type: PropTypes.oneOf(['big', 'small']),
 };
 
 AudioPlayer.defaultProps = {
   width: '100%',
-  height: '165',
-  type: 'small',
-  color: '',
+  height: '360',
```

Comment:
That's the value which will make it look more or less fine both on desktop and mobile devices. However, it would be better if there is a way to detect screen width in js: we could setup height better (will make it smaller on desktops). @Drapegnik do you think it worths implementing?

Check it out: https://dev.wir.by/be/article/kashubskaya-mova

Reply:
https://github.com/haldarmahesh/use-mobile-detect-hook

Reply:
https://usehooks.com/useWindowSize/

Reply:
https://github.com/jaredLunde/react-hook/tree/master/packages/window-size

Reply:
Done. Made height adjust depending on screen's width.